website/docs/concepts/html/fragments.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Fragments"
+---
+
+The `html!` macro always requires a single root node. In order to get around this restriction, you
+can use an "empty tag" (these are also called "fragments").
+
+<!--DOCUSAURUS_CODE_TABS-->
```

Comment:
This is invalid btw. Docusaurus 2 uses Tabs React component in mdx files for tabs in documentation. I take the blame for not converting this when I migrated over v2